src/lib/flight_tasks/tasks/Transition/FlightTaskTransition.cpp

```diff
@@ -64,7 +64,7 @@ void FlightTaskTransition::updateAccelerationEstimate()
 {
 	// Estimate the acceleration by filtering the raw derivative of the velocity estimate
 	// This is done to provide a good estimate of the current acceleration to the next flight task after back-transition
-	_acceleration_setpoint = 0.9f * _acceleration_setpoint + 0.1f * (_velocity - _velocity_prev) / _deltatime;
+	_acceleration_setpoint = .9f * _acceleration_setpoint + .1f * (_velocity - _velocity_prev) / _deltatime;
```

Comment:
@bresch I just saw the problem that this overwrites acceleration setpoints used to level out the vehicle during transition. These setpoints were not executed before and now they actually are. Can I remove this override of the setpoint?

Reply:
I added it for testing: https://github.com/PX4/Firmware/pull/14212/commits/4aaad24d2cdc05117c39720459c9ece4d16eaab2